integration/test/Test/Federation.hs

```diff
@@ -128,7 +128,7 @@ testNotificationsForOfflineBackends = do
 
       -- FUTUREWORK: Uncomment after fixing this bug: https://wearezeta.atlassian.net/browse/WPB-3664
       -- void $ awaitNotification downUser1 downClient1 (Just newMsgNotif) 1 isOtherUser2LeaveUpConvNotif
-      -- void $ awaitNotification otherUser otherClient (Just newMsgNotif) 1 isDelUserLeaveDownConvNotif
+      void $ awaitNotification otherUser otherClient (Just newMsgNotif) isDelUserLeaveDownConvNotif
```

Comment:
Why this change? The test is now failing. Also, it feels like this change doesn't belong to this PR.

Reply:
oh yeah; perhaps I can should revert it, I just looked at the PR  above and it is merged, so the test should pass and I thought I'd just give it a shot and if it doesn't pass, then I comment it again...